### ISSUE TITLE
Colorbars with slightly different range

### ISSUE THREAD

**Issue by Aahelee** [Aahelee/CLIM680#1]
All the subplots have slightly different colorbars. The plot could have been improved if it had the same range in the colorbars for all the subplots. Or just one large colorbar for all subplots.

**Comment by luisortizuriarte** [comment]
You are right that by having diferent colormaps for each panel it's very difficult to compare them against each other. Please look at the documentation for pcolormesh, the function you are using to plot your figures to ensure you have the same boundaries for the colormapping: https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.pcolormesh.html

In particular, take a look at the vmin and vmax options.

We have shown in class how to have a shared colorbar. See our lesson notebook below:
https://github.com/prukutu/clim680-codes/blob/gh-pages/src/multipanel.ipynb

**Comment by luisortizuriarte** [comment]
This issue is solved
